setup.py

```diff
@@ -11,7 +11,7 @@
 
 requirements = [
     # Until COMPAS reaches 1.0, we pin major.minor and allow patch version updates
-    'compas>=0.11,<0.14',
+    'compas>=0.16,<0.17',
```

Comment:
It seems you have a very old base to work on.

Reply:
Yeah I was surprised too. I’ll check what’s up.